CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## X.Y.Z X-Y-Z
+### PaymentSheet
+* [Changed] Enable Intent polling for Revolut Pay
+
```

Comment:
Nit: I'm not sure users will/should know about this change or implementation detail.